packages/wrangler/src/deployment-bundle/apply-middleware.ts

```diff
@@ -41,19 +41,22 @@ export async function applyMiddlewareLoaderFacade(
 		`__MIDDLEWARE_${index}__`,
 		path.resolve(getBasePath(), m.path),
 	]);
+	if (process.env.NODE_ENV === "test") {
```

Comment:
👍

Reply:
took this idea and made a PR with _only_ this! https://github.com/cloudflare/workers-sdk/pull/6043